filament/backend/src/opengl/OpenGLDriver.h

```diff
@@ -113,7 +113,8 @@ class OpenGLDriver final : public backend::DriverBase {
             GLuint id = 0;          // texture or renderbuffer id
             GLenum target = 0;
             GLenum internalFormat = 0;
-            mutable GLsync fence = nullptr;
+            GLuint sidecarRenderBufferMS = 0;  // multi-sample sidecar renderbuffer
```

Comment:
I will do something similar except in my case I will probably have a pointer here instead of a low-level handle, because there are several low-level objects associated with a single texture (VkDeviceMemory, VkImage, VkImageView...).  Since Vulkan handles are 8 bytes, a pointer isn't any worse.